README.md

```diff
@@ -2,4 +2,50 @@
 
 **This is not an official Google product.**
 
-Privacy data mapping and related plans management.
+Pmap is a service for privacy data mapping and privacy plans management to
+conform with Alphabet privacy policies on data classification and inventory.
+
+## Use case
+
+Pmap can be used for users to map specific resources with wipeout plan to adopt
+or maintain privacy policies on data classification and inventory.
+
+## Architecture
+
+![pmap architecture](./docs/assets/pmap-arch.png)
```

Comment:
1. Pmap includes two parts, both resource mapping and wipeout plan. The architecture looks like it only covers resource mapping.
2. Cloud Asset Inventory is just one kind of Processors, make sure the picture shows it clearly. You can refer to the design doc architecture picture
3. For the `User Operation` part, I'd suggesting following https://github.com/abcxyz/jvs#architecture, the separation of `user operation` is not clear as it involves two parts, individual team to register the resource mappings and central privacy team to set up the github repo and github workflow. You don't have to separate `user operation` and `pmap service` here

Reply:
In addition, usually data governor won't query BQ directly. Central privacy team will create a dashboard based on the BQ warehouse and the data governors can view the dashboard.

Reply:
Updated the diagram.